components/movementsensor/movementsensor_test.go

```diff
@@ -263,6 +264,7 @@ var (
 	ang     = spatialmath.AngularVelocity{5.5, 1.2, 2.3}
 	orie    = &spatialmath.EulerAngles{5.6, 1.3, 2.4}
 	compass = 123.
+	la      = r3.Vector{0, 0, 0}
```

Comment:
would it perhaps be better to give this a non-zero value, considering that a zero value is returned in the event of an error or when LinearAcceleration is unimplemented